run.c

```diff
@@ -190,8 +196,9 @@ void softmax(float* x, int size) {
 
 void matmul(float* xout, float* x, float* w, int n, int d) {
     // W (d,n) @ x (n,) -> xout (d,)
-    #pragma omp parallel for
-    for (int i = 0; i < d; i++) {
+    int i;
+    #pragma omp parallel for private(i)
```

Comment:
It doesn't work without the "for private(i)" part in Windows?

Reply:
correct, the counter variable has to be explicitly specified as private to each thread and declared before the pragma unfortunately.